### PR TITLE
Quiz context

### PR DESCRIPTION
**Pull request by jthistle**
https://trello.com/c/NUZYNw3j/98-store-quiz-results-and-display-results-vvv-eee

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/filter**](https://my-best-life-brcvgjdku-neontribe.vercel.app/filter) | [**/quiz/about-you**](https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/about-you) |
|:-:|:-:|
|<a href="https://my-best-life-brcvgjdku-neontribe.vercel.app/filter">Screenshot of /filter</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/SUQoIMGezYebzBelwWbp183HB1c_.png">(view full size)</a> |<a href="https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/about-you">Screenshot of /quiz/about-you</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/QwsqEtixrHooh429i5fto61ByCQ_.png">(view full size)</a> |



| [**/quiz/how-are-you-feeling**](https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/how-are-you-feeling) | [**/quiz/results**](https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/results) |
|:-:|:-:|
|<a href="https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/how-are-you-feeling">Screenshot of /quiz/how-are-you-feeling</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/XLzjwF13g_dowM9v6M9lpXxj0wQ_.png">(view full size)</a> |<a href="https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/results">Screenshot of /quiz/results</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/WZcIk4rqZVysJcI_NmwnseWd_HE_.png">(view full size)</a> |



| [**…hat-are-your-interests**](https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/what-are-your-interests) | [**/quiz/whats-on-your-mind**](https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/whats-on-your-mind) |
|:-:|:-:|
|<a href="https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/what-are-your-interests">Screenshot of /quiz/what-are-your-interests</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Vd_40_4aIU1Nu_12AAittWti5f8_.png">(view full size)</a> |<a href="https://my-best-life-brcvgjdku-neontribe.vercel.app/quiz/whats-on-your-mind">Screenshot of /quiz/whats-on-your-mind</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/uKdge98IbNFFybvkAA9IMUW_ptE_.png">(view full size)</a> |







Commit 13a501829df81a09f2e0f87ed6921ae8ee3103bf (https://my-best-life-brcvgjdku-neontribe.vercel.app).

File: pages/quiz/results.tsx [hunk 1]
```diff
@@ -1,16 +1,120 @@
...
+import { Layout } from '../../src/Components/Layout'
+import { VerticalSpacing } from '../../src/Components/VerticalSpacing'
+import { QuizContext } from '../../src/context/QuizContext'
+import { useRouter } from 'next/router'
+
+interface QuizListPageProps {
+  services: Array<ServicePreview>
+}
+
+export type ServicePreview = Pick<
```
Comment (Nikomus) [inline pages/quiz/results.tsx:16]: This should be importable from ./index

File: pages/quiz/whats-on-your-mind.tsx [hunk 2]
```diff
@@ -86,7 +87,7 @@ export const WhatsOnYourMindPage = (): JSX.Element => {
         />
 
         <QuestionSection>
-          <p>Choose as many as you like</p>
+          <h3>Choose as many as you like</h3>
```
Comment (Nikomus) [inline pages/quiz/whats-on-your-mind.tsx:90]: Do we have an h2 on the page? Heading levels should be sequential

**Comment by jthistle** [comment]
Feedback addressed, just need approval now @Nikomus

**Review by Nikomus — APPROVED** [review]
OK, let's merge this in! 🏆

**Pull request merged**